src/app/modules/item/pages/item-edit/item-edit.component.ts

```diff
@@ -106,15 +137,9 @@ export class ItemEditComponent implements OnDestroy {
       return;
     }
 
-    const itemStringChanges = this.getItemStringChanges();
-    if (!itemStringChanges) {
-      this.errorToast();
-      return;
-    }
-
-    combineLatest([
-      this.updateItemService.updateItem(this.itemId, this.itemChanges),
-      this.updateItemStringService.updateItem(this.itemId, itemStringChanges),
+    forkJoin([
+      this.updateItem(),
+      this.updateString(),
     ]).subscribe(
```

Comment:
In case of failure, the created item info should be kept so that they are not recreated.

Reply:
Done